core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/EventV4Controller.java

```diff
@@ -67,12 +70,14 @@ private StackView getStackViewIfAvailable(String name) {
     }
 
     @Override
-    public StructuredEventContainer structured(String name) {
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
+    public StructuredEventContainer structured(String name, @AccountId String accountId) {
         return legacyStructuredEventService.getStructuredEventsForStack(name, workspaceService.getForCurrentUser().getId());
     }
 
     @Override
-    public Response download(String name) {
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.POWERUSER_ONLY)
```

Comment:
Shouldn't this be allowed for users with access? Or other services have overlays for this?

Reply:
If there is no button on the UI, it's fine for me.